src/services/models/iotHubManagerModels.js

```diff
@@ -167,3 +167,12 @@ export const toNewDeviceRequestModel = ({
         }
   };
 }
+
+export const toDevicePropertiesModel = (iotResponse, dsResponse) => {
+  const dsReported = getItems(dsResponse).map(({ Id }) => Id);
+  const reportedPropertiesUnion = new Set([...iotResponse.Reported, ...dsReported])
+  return {
+      tags: iotResponse.Tags,
```

Comment:
the response from the web service should be a list of "Items" like all the other microservices